package.json

```diff
@@ -58,6 +58,7 @@
     "@thorchain/asgardex-binance": "^3.0.1",
     "@thorchain/asgardex-bitcoin": "^0.3.0",
     "@thorchain/asgardex-crypto": "https://gitlab.com/thorchain/asgardex-common/asgardex-crypto.git",
+    "@thorchain/asgardex-ethereum": "https://gitlab.com/thorchain/asgardex-common/asgardex-ethereum.git#42bada004a43bf6c4b8be60ec33e330e47e84158",
```

Comment:
We will update to latest version ^ as soon as https://gitlab.com/thorchain/asgardex-common/asgardex-ethereum/-/merge_requests/9 has been merged and `asgardex-ethereum` bump to a new version.